src/WalkingModule/include/WalkingControllers/WalkingModule/Module.h

```diff
@@ -285,6 +344,13 @@ namespace WalkingControllers
          * @return true in case of success and false otherwise.
          */
         virtual bool stopWalking() override;
+
+        bool DCMSmoother(const iDynTree::Vector2 adaptedDCM, const iDynTree::Vector2 desiredDCM, iDynTree::Vector2 &smoothedDCM);
+
+        bool FeetTrajectorySmoother(const iDynTree::Transform adaptedFeetTransform, const iDynTree::Transform desiredFootTrajectory,
```

Comment:
Please use a lowercase first letter for the methods. In addition, since I think these are some internal utility, they can be private.

Reply:
Done!